runelite-client/src/main/java/net/runelite/client/plugins/config/PluginHubPanel.java

```diff
@@ -0,0 +1,550 @@
+import net.runelite.client.ui.ColorScheme;
+import net.runelite.client.ui.DynamicGridLayout;
+import net.runelite.client.ui.FontManager;
+import net.runelite.client.ui.PluginPanel;
+import net.runelite.client.ui.components.IconTextField;
+import net.runelite.client.util.ImageUtil;
+import net.runelite.client.util.LinkBrowser;
+import net.runelite.client.util.SwingUtil;
+import net.runelite.client.util.VerificationException;
+import org.apache.commons.text.similarity.JaroWinklerDistance;
+
+@Slf4j
+@Singleton
+class PluginHubPanel extends PluginPanel
+{
+	private static final ImageIcon MISSING_ICON;
+	private static final ImageIcon HELP_ICON;
+	private static final ImageIcon HELP_ICON_HOVER;
+	private static final ImageIcon CONFIGURE_ICON;
+	private static final ImageIcon CONFIGURE_ICON_HOVER;
+	private static final Pattern SPACES = Pattern.compile(" +");
+	private static final JaroWinklerDistance DISTANCE = new JaroWinklerDistance();
+
+	static
+	{
+		BufferedImage missingIcon = ImageUtil.getResourceStreamFromClass(PluginHubPanel.class, "pluginhub_missingicon.png");
+		MISSING_ICON = new ImageIcon(missingIcon);
+
+		BufferedImage helpIcon = ImageUtil.getResourceStreamFromClass(PluginHubPanel.class, "pluginhub_help.png");
+		HELP_ICON = new ImageIcon(helpIcon);
+		HELP_ICON_HOVER = new ImageIcon(ImageUtil.alphaOffset(helpIcon, -100));
+
+		BufferedImage configureIcon = ImageUtil.getResourceStreamFromClass(PluginHubPanel.class, "pluginhub_configure.png");
+		CONFIGURE_ICON = new ImageIcon(configureIcon);
+		CONFIGURE_ICON_HOVER = new ImageIcon(ImageUtil.alphaOffset(configureIcon, -100));
+	}
+
+	private class PluginItem extends JPanel
+	{
+		private static final int HEIGHT = 70;
+		private static final int ICON_WIDTH = 48;
+		private static final int BOTTOM_LINE_HEIGHT = 16;
+		static final float MIN_FILTER_SCORE = .8f;
+
+		private final ExternalPluginManifest manifest;
+
+		@Getter
+		private final boolean installed;
+
+		@Getter
+		private float filter;
+
+		PluginItem(ExternalPluginManifest newManifest, Collection<Plugin> loadedPlugins, boolean installed)
+		{
+			ExternalPluginManifest loaded = null;
+			if (!loadedPlugins.isEmpty())
+			{
+				loaded = ExternalPluginManager.getExternalPluginManifest(loadedPlugins.iterator().next().getClass());
+			}
+
+			manifest = newManifest == null ? loaded : newManifest;
+			this.installed = installed;
+
+			setBackground(ColorScheme.DARKER_GRAY_COLOR);
+			setOpaque(true);
+
+			GroupLayout layout = new GroupLayout(this);
+			setLayout(layout);
+
+			JLabel pluginName = new JLabel(manifest.getDisplayName());
+			pluginName.setFont(FontManager.getRunescapeBoldFont());
+			JLabel author = new JLabel(manifest.getAuthor());
+			author.setFont(FontManager.getRunescapeSmallFont());
+			JLabel version = new JLabel(manifest.getVersion());
+			version.setFont(FontManager.getRunescapeSmallFont());
+			JLabel description = new JLabel(manifest.getDescription());
+			JLabel icon = new JLabel();
```

Comment:
Maybe `JLabel icon = new JLabel("", JLabel.CENTER);` so it look a bit better? 

![1](https://user-images.githubusercontent.com/5454364/70960349-6609ba80-2044-11ea-9410-9fd1b83dca62.png)